src/Analytics.js

```diff
@@ -20,16 +20,17 @@ import PlatformPeg from './PlatformPeg';
 import SdkConfig from './SdkConfig';
 
 function getRedactedUrl() {
-    const base = window.location.pathname.split('/').slice(-2).join('/');
     const redactedHash = window.location.hash.replace(/#\/(room|user)\/(.+)/, "#/$1/<redacted>");
-    return base + redactedHash;
+    // hardcoded url to make piwik happy
+    return 'https://riot.im/app/' + redactedHash;
```

Comment:
Why does this need to be hardcoded? It'll be quite confusing if the URLs all say this even if they're actually on /develop/ or even on a different riot instance